src/Worker.Extensions.DurableTask/DurableTaskFunctionsMiddleware.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Middleware;
+using Microsoft.DurableTask;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.DurableTask;
+
+internal class DurableTaskFunctionsMiddleware : IFunctionsWorkerMiddleware
+{
+    // TODO: Need to add call to action, i.e., pointer to some documentation.
```

Comment:
let's have them also file a bug report if they encounter this and cannot fix it :)